### DISCUSSION TITLE
Installing Winget inside a sandbox

### DISCUSSION THREAD

**Discussion by xenongee** [sandboxie-plus/Sandboxie#3559] [Q&A Feedback]
Can winget-cli be installed inside the Sandboxie sandbox itself? I just tried to install WingetUI, but installing an application from it gave an error:
`Failed to open the predefined source; please report to winget maintainers.
An unexpected error occurred while executing the command:
0x800706d8 :   ()    .`

Upd: Sanboxie gave the notification "cmd.exe: SBIE2205 Service not implemented: ConsoleInit (C00000D4) (3)" (https://sandboxie-plus.com/sandboxie/sbie2205/).
I assume that this is not possible, as Sandboxie does not support non-Win32 applications, ala UWP, but maybe somehow it is possible to give access to the system (outside the sandbox) winget.exe? 

Thanks in advance for the answer!

**Comment by e-t-l** [comment]
Can Winget not be run inside sandboxes to install programs inside sandboxes? Is this a known sbie limitation?
